code/modules/events/immovable_rod.dm

```diff
@@ -60,7 +63,10 @@ In my current plan for it, 'solid' will be defined as anything with density == 1
 			clong.ex_act(2)
 
 	else if (istype(clong, /mob))
+		if(istype(clong, /mob/living/carbon/human))
+			var/mob/living/carbon/human/H = clong
+			H << "The rod penetrates you!"

```

Comment:
uhm

i didn't expect that to work

i theorycrafted this as a joke in piratepad